README.md

```diff
@@ -14,19 +14,44 @@ bulky image viewer ... and don't care about resolution.
 
 ### Install
 
+First get the repository and change into the `src` directory
+
 ```bash
 git clone https://github.com/hzeller/timg.git
 cd timg/src
+```
+
+Compiling on Debian/Ubuntu
```

Comment:
Make these subsections (####) so that it is possible to link to them.

Reply:
done